src/pages/superadmin/header/components/Calendar/App.tsx

```diff
@@ -10,129 +10,132 @@ interface Props {
   setShowCalendar: (show: boolean) => void;
 }
 
+const Section = styled.div`
```

Comment:
we can move these styles to its own CalendarStyles.tsx file and import it here so this tsx file is not as long

Reply:
I can do this right now